Task1/weight.rb

```diff
@@ -0,0 +1,21 @@
+#ideal weight
```

Comment:
fix all rubocop remarks


correct all names in accordance with these recommendations
https://medium.com/selleo/a-subjective-guide-to-naming-stuff-in-ruby-on-rails-classes-b44928b6c49a


the file name should be corresponding with the class name


let's add a call method here and run all other methods inside it

Please read how to use Service Objects in Ruby and try to use them in future tasks. We also use them in rails apps

https://blog.appsignal.com/2020/06/17/using-service-objects-in-ruby-on-rails.html
https://www.honeybadger.io/blog/refactor-ruby-rails-service-object/
https://www.toptal.com/ruby-on-rails/rails-service-objects-tutorial

Service objects are single business actions. You can change the name of your public method if you like. I prefer using call

here is an example
https://paste.ofcode.org/9uJVSsEmVrJZNTTsXs6aLQ